pages/apply.html

```diff
@@ -318,12 +318,28 @@
 
         <div class="usa-input">
 
-          <label class="ss-q-item-label" for="entry_1108678713">What role or team at 18F do you believe you&#39;re the best fit for and why?</label>
-          <p>If you need inspiration, check out <a href="https://pages.18f.gov/joining-18f/who-we-are-hiring/">this page</a>. If you don&#39;t see a perfect role there, that&#39;s ok! Just tell us what you think.</p>
-          <textarea name="entry.1108678713" rows="8" cols="0" class="ss-q-long" id="entry_1108678713" dir="auto" aria-label="What role or team at 18F do you believe you&#39;re the best fit for and why? If you need inspiration, check out this page: https://pages.18f.gov/joining-18f/who-we-are-hiring/. If you don&#39;t see a perfect role there, that&#39;s ok! Just tell us what you think. "></textarea>
+          <label class="ss-q-item-label" for="entry_976680464">Please select the role to which you&#39;d like to apply: <span class="required">*</span></label>
+          <p>For more information about available roles, please refer to <a href="https://pages.18f.gov/joining-18f/who-we-are-hiring/">this page</a>. If you don&#39;t see a role that is a good fit for you, please check back soon as our needs change frequently.</p>
```

Comment:
@fureigh the link in the help text should direct to roles-and-teams (https://pages.18f.gov/joining-18f/roles-and-teams/) instead of who-we-are-hiring. https://pages.18f.gov/joining-18f/roles-and-teams/